sklearn/metrics/_pairwise_distances_reduction/_dispatcher.py

```diff
@@ -265,20 +266,15 @@ def compute(
 
         Notes
         -----
-        This classmethod is responsible for introspecting the arguments
-        values to dispatch to the most appropriate implementation of
-        :class:`ArgKmin64`.
+        This public classmethod is responsible for introspecting the arguments
+        values to dispatch to the private dtype-specialized implementation of
+        :class:`ArgKmin`.
```

Comment:
I would rather not state that this is public because those Cython classes are not part of the scikit-learn public API.

```suggestion
        This classmethod inspects the arguments values to dispatch to the
        dtype-specialized implementation of :class:`ArgKmin`.
```

The docstring of `RadiusNeighbors.compute` should be updated accordingly.